packages/directlinespeech/__tests__/utilities/createTestHarness.js

```diff
@@ -5,11 +5,13 @@ import createQueuedArrayBufferAudioSource from './createQueuedArrayBufferAudioSo
 import fetchSpeechCredentialsWithCache from './fetchSpeechCredentialsWithCache';
 import fetchSpeechData from './fetchSpeechData';
 
-export default async function createTestHarness() {
+export default async function createTestHarness({ 
+  enableInternalHTTPSupport = false }) {
```

Comment:
We should set the default value for the `options` parameter to an empty object.
```suggestion
  enableInternalHTTPSupport = false } = {}) {
```